app/javascript/app/document-capture/services/upload.js

```diff
@@ -0,0 +1,11 @@
+function upload(payload) {
+  return new Promise((resolve, reject) => {
+    const isFailure = window.location.search === '?fail';
+    setTimeout(
+      isFailure ? reject : () => resolve({ ...payload, saved: true }),
+      2000,
```

Comment:
Should we make this configurable? Maybe we could set it in `application.yml` and then expose it as a `<meta>` tag or some sort of `data-` attribute in the DOM?

Reply:
> Should we make this configurable? Maybe we could set it in `application.yml` and then expose it as a `<meta>` tag or some sort of `data-` attribute in the DOM?

I should have probably been clearer, but the implementation of this function is intended as entirely throwaway code. The more important thing is the interface of the function:

```ts
function upload( payload: object ): Promise<object> { /* ... */ }
```

Reply:
Got it! Thanks